src/analysis/inner_product_space/basic.lean

```diff
@@ -1067,6 +1067,19 @@ begin
   simp only [sq, ← mul_div_right_comm, ← add_div]
 end
 
+@[priority 100] -- See note [lower instance priority]
+instance inner_product_space.to_uniform_convex_space : uniform_convex_space F :=
+⟨λ ε hε, begin
+  refine ⟨2 - sqrt (4 - ε^2), sub_pos_of_lt $ (sqrt_lt_iff zero_lt_two).2 _, λ x hx y hy hxy, _⟩,
+  { norm_num,
+    exact pow_pos hε _ },
+  rw sub_sub_cancel,
+  refine le_sqrt_of_sq_le _,
+  rw [sq, eq_sub_iff_add_eq.2 (parallelogram_law_with_norm_real x y), ←sq (∥x - y∥), hx, hy],
```

Comment:
To avoid conflicting with #13498
```suggestion
  rw [sq, eq_sub_iff_add_eq.2 (parallelogram_law_with_norm ℝ x y), ←sq (∥x - y∥), hx, hy],
```

(you'll need to make the arguments to `parallelogram_law_with_norm` explicit like I do there)